webapp-django/crashstats/crashstats/urls.py

```diff
@@ -105,8 +105,7 @@
     url(
         r'^status/$',
         RedirectView.as_view(
-            pattern_name='monitoring:index',
-            permanent=not settings.DEBUG,
```

Comment:
I have no idea why we switch the permanence of this depending on the value of `settings.DEBUG`. That seems unhelpful to me. Also, it was the lone test that didn't work in pytest. I didn't look into why, though. I'm guessing one of a couple of things:

1. pytest bootstrapping is different, so `settings.DEBUG` has a different value at module-load time than what the test is testing for
2. `settings.DEBUG` has a different value when running tests in pytest vs. in nose

Number 1 seems more likely to me.

Anyhow, seemed easier to just ditch this selective permanence.

Reply:
The reason for this was to avoid having a permanent redirect be saved into your browser's cache during development of that feature. It made development easier, but is probably very useless now.